Dockerfile

```diff
@@ -22,4 +22,6 @@ ADD os_wrapper.sh ./os_wrapper.sh
 ADD setup_env ./setup_env
 
 # Running the OS wrapper script
-RUN ./os_wrapper.sh -os ${OS_IMG_NAME} -os-version ${OS_IMG_VERSION}
\ No newline at end of file
+RUN ./os_wrapper.sh -os ${OS_IMG_NAME} -os-version ${OS_IMG_VERSION}
+ADD https://github.com/Yelp/dumb-init/releases/download/v1.1.1/dumb-init_1.1.1_amd64 /usr/local/bin/dumb-init
```

Comment:
Can we fork https://github.com/Yelp/dumb-init/ and then do
```
ADD https://github.com/fabric8-hdd/dumb-init/releases/download/v1.1.1/dumb-init_1.1.1_amd64 /usr/local/bin/dumb-init
```
?

Having a downstream for such dependencies would be much more stable.

Reply:
No we rather rely on their upstream community instead of us maintaining it ourselves DIRECTLY..

Reply:
LGTM for now.
We can continue the discussion on sidelines.